config/windows-clang.cmake

```diff
@@ -0,0 +1,178 @@
+#-----------------------------*-cmake-*----------------------------------------#
+# file   windows-clang.cmake
+# author Kelly Thompson <kgt@lanl.gov>
+# date   2020 March 27
+# brief  Establish flags for Windows - MSVC+Clang
+# note   Copyright (C) 2020 Triad National Security, LLC.
+#        All rights reserved.
+#------------------------------------------------------------------------------#
+
+include_guard(GLOBAL)
+
+# Useful reference information:
+# https://docs.microsoft.com/en-us/cpp/intrinsics/cpuid-cpuidex?view=vs-2017
+
+#
+# Sanity Checks
+#
+if( NOT ${CMAKE_GENERATOR} MATCHES "Visual Studio" AND
+    NOT ${CMAKE_GENERATOR} MATCHES  "NMake Makefiles" )
+  message( FATAL_ERROR
+  "config/windows-cl.cmake must be taught to build for this compiler "
```

Comment:
Should be "config/windows-clang.cmake must..." ?

Reply:
Fixed in aeb6827.